src/libs/checkInternetReachability/index.ts

```diff
@@ -0,0 +1,5 @@
+import type InternetReachabilityCheck from './types';
+
+export default function checkInternetReachability(): InternetReachabilityCheck {
+    return Promise.resolve(true);
```

Comment:
Why is this a no-op for all other platforms? Shouldn't this at least be doing a `NetInfo.fetch()`?